### ISSUE TITLE
Add getfacl and setfacl binaries

### ISSUE THREAD

**Issue by pierreboissinot** [le-phare/le-phare.github.io#128]
Some tasks from [lephare/ansible-deploy](https://github.com/le-phare/ansible-deploy/blob/master/config/steps/set_permission.yml) role require ACL related binaries: `setfacl` and `getfacl`.

These tasks are skipped by default: https://github.com/le-phare/ansible-deploy/blob/641c82192a8efeffd2ac0abfc6c88f7997c82d73/defaults/main.yml#L63

**Comment by thislg** [comment]
Ce n'est pas activé par défaut et l'option est très peu utilisée donc je ne suis pas convaincu du besoin de mettre ça en prérequis.

**Comment by pierreboissinot** [comment]
> Ce n'est pas activé par défaut et l'option est très peu utilisée donc je ne suis pas convaincu du besoin de mettre ça en prérequis.

Oui, ceci dit l'impact est faible; ce n'est pas comme si on installait un elasticsearch par défaut.

Ca ne me dérange pas car j'ajoute ce genre de binaires "optionnels" dans les pré-requis que j'envoie; autrement dit c'est la responsabilité du lead dev.

@caillaudv tu es d'accord avec @thislg  ?

**Comment by pierreboissinot** [comment]
@thislg je vous propose qu'on mentionne dans dans le readme ok pr vous ?
